### ISSUE TITLE
About fine-tune

### ISSUE THREAD

**Issue by lelimite4444** [mrharicot/monodepth#150]
Dear Author,

I'm now implementing your work in PyTorch and I'm going to fine-tune on KITTI.
And I'd like to know your settings for fine-tuning on the model pretrained on CityScapes,
like learning rate, epoch etc.

Thank you

**Comment by mrharicot** [comment]
Hi,
We use the exact same parameters as with KITTI, we only keep the top 4/5th of the image thus removing the car hood from the image.

**Comment by Yihanhu** [comment]
Dear Author,
I wonder how many epoch you pretrained on cityscapes and how many epoch you fine-tuned on kitti. Also, what is your learning rate Strategy? Thank a lot!

**Comment by mrharicot** [comment]
Hi,
The pretraining and fine tuning procedures are identical to the one we describe in the paper:
50 epochs with an initial learning rate of 1e-4 which we halve after 30 and 40 epochs.

**Comment by Yihanhu** [comment]
Thanks for you reply!
Got it, you mean you first train 50 epoch on Cityscapes and then 50 epoch on KITTI, right?

**Comment by mrharicot** [comment]
Yep!
